docroot/sites/all/themes/custom/wsdot_classic/css/styles.css

```diff
@@ -821,6 +821,7 @@ div.workbench-info-block {
 /* YouTube player */
 a.play-button {position:absolute;top:50%;left:50%;width:100px;height:100px;margin-top:-50px;margin-left:-50px;text-indent: 100%;white-space: nowrap;overflow: hidden;background:url(../images/play.png) no-repeat;background-size:100%;}
 .youtube-container {display:block;width:510px;max-width:98%;margin:2em 0;}
+.panels-flexible-region-52-center .youtube-container {width:450px;}
```

Comment:
Where is the `.panels-flexible-region-52-center` class currently being referenced on the site? I'm not seeing it anywhere.